features/admin.sms-providers.v1/pages/twilio-sms-provider.tsx

```diff
@@ -42,6 +43,24 @@ const TwilioSMSProvider: FunctionComponent<TwilioSMSProviderPageInterface> = (
         isReadOnly
     } = props;
     const { t } = useTranslation();
+    const [ isShow, setIsShow ] = useState(false);
+
+    const renderInputAdornment = (): ReactElement => {
```

Comment:
Can use shorthand here.

```suggestion
    const renderInputAdornment = (): ReactElement => (
        <InputAdornment
            ...
```

Reply:
Improved with d889f21bccc2eabbe52d21f21297bcd144289f5b.